docs/index.rst

```diff
@@ -91,6 +91,30 @@ using ``iris-grib``::
     >>> iris.save(my_cube, 'my_file.grib2')
 
 
+Getting Started
+===============
+
+To ensure all ``iris-grib`` dependencies, it is sufficient to have installed
+:mod:`Iris <iris>` itself, and
+`ecCodes <https://software.ecmwf.int/wiki/display/ECC/ecCodes+Home>`_ .
+
+The simplest way to install is with
+`conda <https://conda.io/miniconda.html>`_ ,
+using the `conda-forge channel <https://anaconda.org/conda-forge>`_ ,
+with the command
+
+    $ conda install iris-grib -c conda-forge
```

Comment:
I'm still not quite sure what our position is with respect to advising people to use conda.  I'll have a chat with Pete about this I suppose.

Reply:
> have a chat with Pete

you mean @dkillick ;-) 
Good idea. 
Well, Iris docs has gone over to pip, but its README still recommends conda. 
I certainly still think conda is most reliable + I wouldn't use anything else for choice myself.
IMO this ought to be fine, as (a) it's for external users and (b) we don't provide a pip install for iris-grib anyway (though perhaps we should, if that's the way it's going).

Reply:
Yeah sorry, I will have a chat with @dkillick because we are virtual people now.

Reply:
I actually don't exist.

P.S. I think saying "use conda" is fine, and we [recommend it for Iris](https://scitools.org.uk/iris/docs/latest/installing.html#installing-using-conda).

Reply:
> we don't provide a pip install for iris-grib ... (though perhaps we should)

I think we should. Many other SciTools packages are now on PyPI, and we probably should support it as PyPI remains popular.

Reply:
Ok, but I don't think it needs to be done right now.
#14

Reply:
> I don't think it needs to be done right now

Agreed. Thanks for raising an issue about it.